### PR TITLE
Add CubicFeetPerPound unit in Specific Volume.

### PR DESCRIPTION
**Pull request by mp9007**

### CODE REVIEW

File: Common/UnitDefinitions/SpecificVolume.json [hunk 1]
```diff
@@ -19,6 +19,19 @@
           "Abbreviations": [ "m³/kg" ]
         }
       ]
+    },
+    {
+      "SingularName": "CubicFeetPerPound",
+      "PluralName": "CubicFeetsPerPounds",
+      "FromUnitToBaseFunc": "x/0.062427959968012",
+      "FromBaseToUnitFunc": "x*0.062427959968012",
```
Comment (gojanpaolo) [inline Common/UnitDefinitions/SpecificVolume.json]: Is the conversion the other way around?

1 ft³/lb = 0.062428 m³/kg

**Comment by gojanpaolo** [comment]
@mp9007 Thanks for the PR. Please see comments for review :)

**Comment by mp9007** [comment]
I updated the wording too.

File: Common/UnitDefinitions/SpecificVolume.json [hunk 2]
```diff
@@ -21,15 +21,15 @@
       ]
     },
     {
-      "SingularName": "CubicFeetPerPound",
-      "PluralName": "CubicFeetsPerPounds",
-      "FromUnitToBaseFunc": "x/0.062427959968012",
-      "FromBaseToUnitFunc": "x*0.062427959968012",
+      "SingularName": "CubicFootPerPound",
+      "PluralName": "CubicFeetPerPounds",
```
Comment (gojanpaolo) [inline Common/UnitDefinitions/SpecificVolume.json]: @mp9007 Kindly rename PluralName to `CubicFeetPerPound`. :D We keep the `PerPound` singular.
Aside from that, everything looks good +1

Comment (angularsen) [reply Common/UnitDefinitions/SpecificVolume.json]: @mp9007 Just a reminder that we are waiting for you to make this change. Also run `generate-code.bat` once more afterwards.

**Comment by angularsen** [comment]
@mp9007 Hi, just a friendly reminder that we are waiting for you here. It would be good to know if you still intend to follow up on it.

**Comment by gojanpaolo** [comment]
@angularsen Please review. :) Will release nuget after closing this PR.

**Comment by angularsen** [comment]
There is a build error though, see appveyor link.

```
CustomCode\SpecificVolumeTests.cs(53,35): error CS0115: 'SpecificVolumeTests.CubicFeetPerPoundsInOneCubicMeterPerKilogram': no suitable method found to override [C:\projects\unitsnet\UnitsNet.Tests\UnitsNet.Tests.csproj]
CustomCode\SpecificVolumeTests.cs(48,18): error CS0534: 'SpecificVolumeTests' does not implement inherited abstract member 'SpecificVolumeTestsBase.CubicFeetPerPoundInOneCubicMeterPerKilogram.get' [C:\projects\unitsnet\UnitsNet.Tests\UnitsNet.Tests.csproj]
```

**Comment by gojanpaolo** [comment]
Fixed. Sorry about that.

**Comment by angularsen** [comment]
Nice, thanks!

**Pull request merged**